src/visualization/write2vtk.jl

```diff
@@ -194,6 +195,7 @@ function trixi2vtk(coordinates; output_directory="out", prefix="", filename="coo
     vtk_grid(file, points, cells) do vtk
         # Store particle index.
         vtk["index"] = [i for i in axes(coordinates, 2)]
+        vtk["ndims"] = size(coordinates, 1)
```

Comment:
```suggestion
        vtk["ndims"] = size(coordinates, 1)
        vtk["particle_spacing"] = particle_spacing
```
add the keyword `particle_spacing=-1` above.
Plus, add the particle spacing in the function in line ~230:
```julia
function trixi2vtk(initial_condition::InitialCondition; output_directory="out",
                   prefix="", filename="initial_condition", custom_quantities...)
    (; coordinates, velocity, density, mass, pressure) = initial_condition

    return trixi2vtk(coordinates; output_directory, prefix, filename,
                     density=density, initial_velocity=velocity, mass=mass,
                     particle_spacing=initial_condition.particle_spacing,
                     pressure=pressure, custom_quantities...)
end
```